internal/common/common.go

```diff
@@ -393,6 +393,16 @@ func GetTagFromImageRef(ref string) string {
 	}
 }
 
+func GetAPIVipDNSName(c *Cluster) string {
+	//In case cluster that is not None platform and api vip is set we should set api vip as APIVipDNSName
+	// I know that APIVipDNSName name is confusing but in order not to add new variable we will use it
+	// as we allow already setting ip to it
+	if !swag.BoolValue(c.Cluster.UserManagedNetworking) && c.Cluster.APIVip != "" {
```

Comment:
Unsure we can trust UMN, but I guess if we have the API VIP we can just use it

Reply:
i prefer to verify both, in SNO case we will set api_vip but we still need to use dns name